File: backend/app/schemas/stats.py
from pydantic import BaseModel


class ACWRResponse(BaseModel):
    acute_load: float
    chronic_load: float
    acwr_ratio: float | None
    risk_zone: str
    is_calibrating: bool
    session_count: int


class DailyVolumePoint(BaseModel):
    date: str
    total_load: float
    total_duration: int

File: frontend/src/types/stats.ts
export interface ACWRResponse {
  acute_load: number
  chronic_load: number
  acwr_ratio: number | null
  risk_zone: string
  is_calibrating: boolean
}

export interface DailyVolumePoint {
  date: string
  total_load: number
  total_duration: number
}
